std/format.d

```diff
@@ -2948,6 +2953,18 @@ useSnprintf:
            ~"0000000000000000000000000000000000000000000000000000000000000000000p+0");
 }
 
+@safe unittest
```

Comment:
I guess this can't be `pure` because of the floats?

Reply:
Surprisingly (for my) it can be pure, while the quite similar test directly above cannot. The `static` seems to make a difference here.